content-src/components/DiscoveryStreamComponents/DSImage/DSImage.jsx

```diff
@@ -42,7 +42,13 @@ export class DSImage extends React.PureComponent {
   }
 
   componentDidMount() {
-    this.observer = new IntersectionObserver(this.onSeen.bind(this));
+    this.observer = new IntersectionObserver(this.onSeen.bind(this), {
+      // Assume an image will be eventually seen if it is within 520px of the viewport
+      // This is half the average Desktop vertical screen size:
+      // http://gs.statcounter.com/screen-resolution-stats/desktop/north-america
+      rootMargin: `520px`,
```

Comment:
With this patch, we are increasing images we are loading on first run for higher resolutions. Not sure about 520px number here. Looking at stat counter it shows 1920x1080 and 1366x768 as the most used. Changing Mac to 1680x1050 shows hard coded layout loading 9 images.
@ncloudioj do we have telemetry numbers of whats most favored screen resolution for Firefox users

Reply:
Yes, we do. Check out the `Display resolution` on [this page](https://data.firefox.com/dashboard/hardware).

Reply:
"Two display resolutions, 1366x768px and 1920x1080px, stand out as the most highly-used, with the latter showing an upward trend"

@punamdahiya So it seems like `540px` would be slightly more accurate given our numbers. Is that cool?

Reply:
@gvn I am good with 540px, patch definitely optimize loading images early, I am not sure if jump from 3 to 9 images on first load has any performance number shift. Only way to overrule that is run some recordings and verify which I am ok doing in a separate bug about running performance numbers for 69 release